pkg/kubelet/rkt/rkt.go

```diff
@@ -114,21 +127,16 @@ func New(config *Config,
 	imageBackOff *util.Backoff,
 	serializeImagePulls bool,
 ) (*Runtime, error) {
-	systemdVersion, err := getSystemdVersion()
-	if err != nil {
-		return nil, err
-	}
-	result, err := systemdVersion.Compare(systemdMinimumVersion)
+	// Create dbus connection.
+	systemd, err := newSystemd()
```

Comment:
I don't know what's happening here, but if you click the second commit, it shows up
https://github.com/yifan-gu/kubernetes/commit/da3a66eaa9fadb8d535b879360f0b5873a1f53dd